docker-compose.yml

```diff
@@ -19,19 +19,32 @@ services:
       - ./volume:/volume
     ports:
       - "8080:8080"
+    # starting with blindly following example, this flag has pretty big security impls  
```

Comment:
the example repo has this set to true, but we can look into having a more minimal configuration in the future. see: https://fly.io/docs/litefs/getting-started-docker/#docker-container-privileges